src/test/java/steps/DropdownSteps.java

```diff
@@ -26,23 +26,23 @@ public void user_is_on_the_dropdown_page(){
     }
 
     @When("user clicks the menu and selects {string}")
```

Comment:
try implementing step def where you can pass name of the element and value to be selected - Try using "Reflections" in java for this

Reply:
You mean passing both name and value on the same line?